pkgs/development/python-modules/pykka/default.nix

```diff
@@ -1,27 +1,20 @@
-{ stdenv
-, buildPythonPackage
-, fetchFromGitHub
-}:
+{ lib, buildPythonPackage, fetchPypi, pytestCheckHook, pytest-mock }:
 
 buildPythonPackage rec {
-  pname = "pykka";
-  version = "2.0.1";
+  pname = "Pykka";
```

Comment:
Does this require a rename in all-packages?

Reply:
I think it's not necessary since it would _just_ change the derivation name

Reply:
it's not needed. Attrs shoudl now follow PEP 0503

According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)